clients/flash/air-client/src/css/xhdpi.css

```diff
@@ -106,9 +106,17 @@
 		fontSize     : 22;
 	}
 	
-	poll|PollButtons {
-		padding      : 12;
-		gap          : 18;
+	main|MobilePopUp {
+		cornerRadius : 8;
+		fontSize     : 22;
+	}
+	
+	main|MobilePopUp {

```

Comment:
These two "main|MobilePopUp" definitions should be merged together. Change needs to be applied to all the other profiles also.